pkg/registries/registries.go

```diff
@@ -14,6 +14,12 @@ func scopeMatchesRegistry(scope, reg string) bool {
 	if reg == scope {
 		return true
 	}
+	if isValidWildcard(reg) {
```

Comment:
`scope` is defined to have the Prefix format; `reg` is supposed to be a host:port. So checking for wildcard in `reg` does not make sense, at least in how that function is currently named and documented. See elsewhere about `scopeIsSubscopeOf`.